Makefile

```diff
@@ -712,6 +712,15 @@ test-go-unit:
 		| tee $(TEST_LOG_DIR)/unit.json \
 		| gotestsum --raw-command -- cat
 
+# Runs tbot unit tests
+.PHONY: test-go-unit-tbot
+test-go-unit-tbot: FLAGS ?= -race -shuffle on
+test-go-unit-tbot: SUBJECT ?= ./tool/tbot/... ./lib/tbot/...
+test-go-unit-tbot:
+	$(CGOFLAG) go test -cover -json $(PACKAGES) $(SUBJECT) $(FLAGS) $(ADDFLAGS) \
```

Comment:
What is PACKAGES in this context? Aren't lib/tbot and tool/tbot the packages we're testing?

Reply:
e7f021ed01

Reply:
At this point, do we even need `SUBJECT`? Can we just inline the packages right in the `go test` command?

Reply:
[8f4518e](https://github.com/gravitational/teleport/pull/31093/commits/8f4518e4f7835b2d013ae6e961184c5466de26f5)